app/views/prior_authority/applications/show.html.erb

```diff
@@ -33,7 +56,8 @@
       <h2 class="govuk-heading-m"><%= section_group[:heading] %></h2>
       <% section_group[:sections].each do |section| %>
         <% if section.template %>
-          <%= render(section.template, section: section, show_actions: false) %>
+          <% allowance_type = { 'granted' => :original, 'part_grant' => :dynamic, 'rejected' => :na }.fetch(@application.status, false) %>
+          <%= render(section.template, section: section, show_actions: false, show_allowances: allowance_type) %>
```

Comment:
Naming wise might `show_adjustment[s]` match the overall logic. In my head, having been looking at adjustments in case worker a lot recently, that currently makes more sense to me but maybe i cannot see the wood for the trees.

```suggestion
          <%= render(section.template, section: section, show_actions: false, show_adjustment: adjustment_type) %>
```